iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -92,380 +71,355 @@ TEST_F(ServiceDiscovery_test, FindServiceWithWildcardsReturnsOnlyIntrospectionSe
     }
 }
 
-TEST_F(ServiceDiscovery_test, OfferSingleMethodServiceSingleInstance)
+TEST_F(ServiceDiscovery_test, FindServiceReturnsOfferedService)
 {
     ::testing::Test::RecordProperty("TEST_ID", "30f0e255-3584-4ab2-b7a6-85c16026852d");
-    auto isServiceOffered = sut.offerService({"service1", "instance1", "event1"});
-    this->InterOpWait();
+    const iox::capro::ServiceDescription SERVICE_DESCRIPTION("service", "instance", "event");
```

Comment:
In general you could think about making the strings passed to the `ServiceDescription` unique. I always have a better feeling, because in theory due to some error the old `ServiceDescription` of another test case could still be in the memory and could potentially cover over a failing test. No hard feelings here, but I'm sure Hypnotoad would appreciate :grin:

Reply:
I expect we have tests of the `ServiceDescription`, `UntypedPublisher`, etc. that guarantee a correct clean up.